<launch>
  <!-- SIMULATION -->
    <include file="$(find turbot)/launch/simulation.launch">
      <arg name="enableImu"   value="true"/>
      <arg name="enableDepth" value="true"/>
      <arg name="enableDvl"   value="true"/>
      <arg name="enableGps"   value="true"/>
      <arg name="enableModem" value="true"/>
      <arg name="enableVO"    value="false"/>
      <arg name="enableUWSim" value="true"/>
    </include>






  <!-- PLOT TRAJECTORIES -->
  <!-- gps  -->
<!--   <node pkg="usbl_position" name="plot_path_gps" type="plot_path">
    <param name="plot_path_gps/topic_name"   value="/sensors/gps" />
    <param name="plot_path_gps/sensor"   value="/gps" />
    <param name="plot_path_gps/id"   value="1" />
  </node> -->
  <!-- usbl delayed-->
<!--   <node pkg="usbl_position" name="plot_path_usbl" type="plot_path">
    <param name="plot_path_usbl/topic_name"   value="/sensors/modem_delayed" />
    <param name="plot_path_usbl/sensor"   value="/modem_delayed" />
    <param name="plot_path_usbl/id"   value="2" />
  </node>  -->
  <!-- usbl -->
  <node pkg="usbl_position" name="plot_path_usbl_up" type="plot_path">
    <param name="plot_path/topic_name"   value="/sensors/modem_raw" />
    <param name="plot_path/sensor"   value="/modem_raw" />
    <param name="plot_path/id"   value="1" />
  </node> 

  <!-- RVIZ -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find usbl_position)/config/plotting.rviz" /> 

</launch>

